bisulfite/bam_to_trimmed_fastq_and_biscuit_alignments.cwl

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "bam to trimmed fastqs and biscuit alignments"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: InlineJavascriptRequirement
```

Comment:
It doesn't look like this currently uses the `InlineJavascriptRequirement` or the `MultipleInputFeatureRequirement`.